packages/owl-bot/src/should-ignore-repo.ts

```diff
@@ -32,6 +32,6 @@ export function shouldIgnoreRepo(ownerSlashRepo: string): boolean {
   // subdirectories of google-cloud-php and never accept pull requests.
   const regexp = process.env.IGNORE_REPO_REGEXP
```

Comment:
Seems like these should always be ignored, not only if they're not explicitly specified.

Reply:
I think is an override for tests/testing environments and we use the default in production